README.md

```diff
@@ -341,6 +343,51 @@ This section describes what you need to change to make your code compile with `r
    + Use `(putStr[Ln]|readFile|writeFile|appendFile)[Text|LText|BS|LBS]` functions.
 6. Run `hlint` using `.hlint.yaml` file from `relude` package to cleanup code and imports.
 
+## Comparison with other alternative preludes [↑](#structure-of-this-tutorial)
+
+There are quite a few libraries that can be used as alternative preludes in
+Haskell, let's compare Relude with some of them.
+
+### Relude vs Protolude [↑](#structure-of-this-tutorial)
+
+[Protolude](https://github.com/sdiehl/protolude) is one of the most popular
+alternative preludes. It's also relatively small, but:
+
+1. Protolude supports older GHC versions (from GHC 7.6.1) while `relude` only
+   supports from GHC 8.0.2. So if you aim ancient GHC versions, `protolude`
+   might be a better choice. But because of that it contains a lot of CPP, code
+   is ugly in some places as a consequence and it's more difficult to add,
+   remove or change things there.
+2. `relude` has much better documentation:
+    * [High-level overview of internal module structure](http://hackage.haskell.org/package/relude/docs/Relude.html)
+    * 100% Haddock coverage
+    * Almost every function has usage examples and all examples are tested with
+      `doctest` (which also sometimes hard to do because of multiple GHC
+      versions support, but we try really hard)
+    * [Tutorial + migration guide](structure-of-this-tutorial) from
+      `Prelude` and just general description of the whole package and libraries
+      it depends on.
+3. `relude` has custom HLint rules specific to it: you can use them to remove
+   redundant imports or find hints how to use functions from `relude`. Moreover,
+   the HLint rules are generated using Dhall and there is [a blog post about
+   used technique](https://kowainik.github.io/posts/2018-09-09-dhall-to-hlint).
```

Comment:
Maybe `a blog post about this technique` instead? sounds a bit more fluid to me at least